### PR TITLE
Support Hack root in an arbitrary subdirectory

### PR DESCRIPTION
**Pull request by bigfootjon**
Closes: #94 

This didn't end up being as easy as I hypothesized in the issue, but this seems like the best way to support a non-workspace root for Hack code.

### CODE REVIEW

**Comment by PranayAgarwal** [comment]
I'm a bit hesitant to ship this because people have already complained about too many confusing path configurations in this extension, and this adds one more in the mix

File: package.json [hunk 1]
```diff
@@ -196,6 +196,11 @@
           "default": "hh_client",
           "description": "Absolute path to the hh_client executable. This can be left empty if hh_client is already in your environment $PATH."
         },
+        "hack.rootPath": {
```
Comment (PranayAgarwal) [inline package.json:199]: Could you instead change this to be a complete path to the `.hhconfig` file itself?

**Comment by bigfootjon** [comment]
@PranayAgarwal, sorry this took me ~8 months to respond to your changes, I got sidetracked on other side projects :/

Please take another look when you get a chance

**Comment by bigfootjon** [comment]
I no longer use HHVM so I'm going to close this

**Pull request closed without merging**